docs/user/how-to.md

```diff
@@ -0,0 +1,13 @@
+# How-to guides
+
+This section contains some how-to guides for common tasks.
+
+## How do I paste to a target?
+
+To replace a target with the contents of the clipboard, say `"paste to <target>"`. For example, `"paste to air"`.
```

Comment:
Suggestion: Through, lead with the raw example. So:

> `"paste to air"`
>
> To replace a target with the contents of the clipboard, say `"paste to <target>"`.

(https://en.wikipedia.org/wiki/BLUF_(communication))

Reply:
Also, suggestion: make each spoken form word link to reference.

`paste` -> reference for `paste`
`to` -> reference for destination
`air` -> reference for targets, which starts with "air is the generic target used in examples" and a link to talon alphabet and then more

(Or does `paste to` link to ref for `paste`?)

Reply:
> Suggestion: Throughout, lead with the raw example. So:
> 
> > `"paste to air"`
> > To replace a target with the contents of the clipboard, say `"paste to <target>"`.
> 
> (https://en.wikipedia.org/wiki/BLUF_(communication))

done.

> Also, suggestion: make each spoken form word link to reference.
> 
> `paste` -> reference for `paste` `to` -> reference for destination `air` -> reference for targets, which starts with "air is the generic target used in examples" and a link to talon alphabet and then more
> 
> (Or does `paste to` link to ref for `paste`?)

This is a nice idea, but a) I'm not sure it will be clear that each separate word is a link, and b) I'm not sure exactly how to do that within an inline code snippet. I guess I can not use inline code snippet here. We could prob create a special element we use for all voice commands. And we might even use a different one for voice commands that are just part of a command vs a full command you can say

Another issue is that linking to "paste" is brittle right now, because it's just a header. If we use proper pages for each action / scope / whatever, then vscode will automatically update references if I move a file

Reply:
> I'm not sure it will be clear that each separate word is a link

If it is underlined, I think it'll be clear enough?

All reasonable concerns. We can revisit later as desired; retrofitting this in would be only mildly annoying.

Reply:
https://github.com/cursorless-dev/cursorless/issues/2232